introduction.md

```diff
@@ -0,0 +1 @@
+Hi dojo, my name is Sandy, I am busy with the course to master Github.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.